platform/remarkable/enable-rm2-wifi.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if lsmod | grep -q brcmfmac; then
+    ifconfig wlan0 up
+else
+    modprobe brcmfmac
+fi
+
+# clean out any other running instances of wpa_supplicant
+killall wpa_supplicant 2>/dev/null
```

Comment:
`wpa_cli terminate` ought to be able to do that, hopefully in a cleaner manner ;).

Reply:
Hmm, the rm2's default software runs wpa_supplicant as a DBus service, which wpa_cli doesn't connect to.
But you're right, I do prefer "cleaner" over "simpler" if they're at odds.
So replacing with `systemctl stop wpa_supplicant` AND `wpa_cli terminate`.
Both of them run in a tiny fraction of a second, and both of them behave fine regardless of which flavor of wpa_supplicant is or isn't running.

Reply:
> (Patches to plug the KO UI into platform system configs welcome ;p. It was never done in order to keep the client platform-agnostic, I think. And then we kinda ruined that on Kobo by having scripts that half bypass it ;p).

I definitely understand living with and/or hacking workarounds for old architectural decisions :)

Still getting a hang of the overall project structure. I'll probably work up a patch when I decide the current situation bugs me enough ;P